app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -274,8 +274,8 @@ class MainActivity : ComponentActivity() {
         }
     }
 
-    fun hasCameraPermission() = ContextCompat.checkSelfPermission(this, Manifest.permission.CAMERA)
-    fun hasExternalStoragePermission() = ContextCompat.checkSelfPermission(this, Manifest.permission.WRITE_EXTERNAL_STORAGE)
+    private fun hasCameraPermission() = ContextCompat.checkSelfPermission(this, Manifest.permission.CAMERA)
+    private fun hasExternalStoragePermission() = ContextCompat.checkSelfPermission(this, Manifest.permission.WRITE_EXTERNAL_STORAGE)
```

Comment:
Implemented data hiding on these functions as they are only being used in MainActivity,kt

Reply:
Good.